front/app/components/FormBuilder/components/FormBuilderSettings/ConfigSelectWithLocaleSwitcher/index.tsx

```diff
@@ -92,7 +93,7 @@ const ConfigSelectWithLocaleSwitcher = ({
   };
 
   const defaultOptionValues = [{}];
-  const errors = formContextErrors[name] as RHFErrors;
+  const errors = get(formContextErrors, name) as RHFErrors;
```

Comment:
In a PR about 3 weeks ago, the lodash "get" was removed as it was something discussed in the FE meeitng, but in the case of the React Hook Form, we actually do need them, since the name attribute can be in dot format (e.g. customFields.2.title_multiloc). So I've added them all back in for now (checked with Iva as well).